common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_backhaul.yaml

```
@@ -61,6 +61,14 @@ cACTION_BACKHAUL_ENABLE:
   backhaul_preferred_radio_band: uint8_t
   frequency_band: beerocks::eFreqType
   max_bandwidth: beerocks::eWiFiBandwidth 
+  ht_capability: uint16_t
+  ht_mcs_set:
+    _type: uint8_t
+    _length: [ "beerocks::message::HT_MCS_SET_SIZE" ]
```

```suggestion
    _length: [ beerocks::message::HT_MCS_SET_SIZE ]
```
Quoting is not needed.

---

I get the following error if not quoting:
```
-- Generating Beerocks TLVF files...
Traceback (most recent call last):
  File "/home/developer/workspace/framework/tlvf/tlvf.py", line 1999, in <module>
    main()
  File "/home/developer/workspace/framework/tlvf/tlvf.py", line 1996, in main
    tlvf = TlvF(args.src_path, args.yaml_path, args.out_path, args.conf, args.print_dependencies, args.print_outputs)
  File "/home/developer/workspace/framework/tlvf/tlvf.py", line 404, in __init__
    self.loadAllYamlFilesToDB()
  File "/home/developer/workspace/framework/tlvf/tlvf.py", line 1799, in loadAllYamlFilesToDB
    yaml_inst = self.loadYaml(fname)
  File "/home/developer/workspace/framework/tlvf/tlvf.py", line 1919, in loadYaml
    raise Exception("  %s" % str(exc))
Exception:   while scanning a plain scalar
  in "/home/developer/workspace/common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_backhaul.yaml", line 67, column 16
found unexpected ':'
  in "/home/developer/workspace/common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_backhaul.yaml", line 67, column 24
Please check http://pyyaml.org/wiki/YAMLColonInFlowContext for details.
CMake Error at common/beerocks/tlvf/CMakeLists.txt:28 (message):
  -- /home/developer/workspace/framework/tlvf/tlvf.py --print-outputs failed!
```

---

@LiorAmram please comment - I think its a bug in `tlvf.py` which doesn't generate this properly

---

From what I see in other yaml files it looks like length variables are unquoted when it's defined in the yaml file itself, while they are quoted when they are defined in header (C++) files.

I wonder why the length is always a one-element list though.